SCOPES.md

```diff
@@ -0,0 +1,45 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:beetmover`
+  * Used for Firefox/Fennec/Devedition style beetmover tasks
+
+* `project:comm:thunderbird:releng:beetmover`
+  * Used for Thunderbird style balrog tasks
+
+
+# Scopes
+* `{scope_prefix}:action:push-to-candidates`
+  * Tells beetmoverscript to Schedule a release to ship on balrog channel(s)
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-candidate-tasks`
+
+* `{scope_prefix}:action:push-to-release`
+  * Tells beetmoverscript to submit locale information for a release.
+    * This is **default** if no `action` specified in scopes.
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-push-tasks`
+
+* `{scope_prefix}:action:push-to-nightly`
+  * Tells beetmoverscript to submit the initial toplevel release information.
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-nightly-tasks`
+
+* `{scope_prefix}:bucket:release`
+  * Tells beetmoverscript to use the aurora bucket config
+    * *Informative*: This is production balrog, with aurora channel (devedition)
+  * **Conflicts**: with any other `{scope_prefix}:bucket:*`
+  * **Branch Restrictions**: `all-release-branches`
+
+* `{scope_prefix}:bucket:nightly`
+  * Tells beetmoverscript to use the nightly bucket config
+    * *Informative*: This is production balrog, with nightly channel
+  * **Conflicts**: with any other `{scope_prefix}:bucket:*`
+  * **Branch Restrictions**: 
+    * `all-nightly-branches`
+
+* `{scope_prefix}:bucket:dep`
+  * Tells beetmoverscript to use the nightly bucket config
+    * *Informative*: This is staging balrog, with any (otherwise valid) channel
+  * **Conflicts**: with any other `{scope_prefix}:bucket:*`
+  * **Branch Restrictions**: `None`
+
```

Comment:
There are also another two types of scopes missing here, from [puppet](https://hg.mozilla.org/build/puppet/file/tip/modules/beetmover_scriptworker/templates/prod_script_config.json.erb)

`partner` - "Tells beetmoverscript to use the production credentials to access the `partner` bucket"
`dep-partner` - same as above but for staging